www/modules/lulu.js

```diff
@@ -0,0 +1,29 @@
+
+class LuluModule extends Module {
+	constructor (params) {
+		super ("lulu", "https://github.com/tobiasgf/lulu");
```

Comment:
Please add an help for your module, not for the software.
See other previously added modules for an example.